web/src/components/metrics/histogram.tsx

```diff
@@ -113,17 +130,74 @@ export const Histogram: React.FC<{
     [moveHistogramRange, moveRange, zoomRange]
   );
 
-  const zoomButtonTips = () => {
+  const zoomButtonTips = React.useCallback(() => {
     return t('Zoom in / out histogram. You can also use plus or minus buttons while histogram is focused.');
-  };
+  }, [t]);
 
-  const pageButtonTips = () => {
+  const pageButtonTips = React.useCallback(() => {
     return t('Move selected range. You can also use page up or down buttons while histogram is focused.');
-  };
+  }, [t]);
 
-  const arrowButtonTips = () => {
+  const arrowButtonTips = React.useCallback(() => {
     return t('Move displayed range. You can also use arrow left or right buttons while histogram is focused.');
-  };
+  }, [t]);
+
+  const [guidedTourDone, setGuidedTourDone] = useLocalStorage<boolean>(LOCAL_STORAGE_HISTOGRAM_GUIDED_TOUR_DONE_KEY);
+  React.useEffect(() => {
+    if (!guidedTourHandle) {
+      return;
+    }
+
+    guidedTourHandle.updateTourItems([
+      {
+        title: t('Histogram'),
+        description: t(
+          // eslint-disable-next-line max-len
+          'The following bar chart represent the number of logs across time. You can select a portion of it to drill down into the selected time range and filter the content below accordingly.'
```

Comment:
typo "represents":
```suggestion
          'The following bar chart represents the number of logs across time. You can select a portion of it to drill down into the selected time range and filter the content below accordingly.'
```

Also, I would rather say "over time" than "across time", but defer on native speakers on that, cc @stleerh @skrthomas

Reply:
I agree with over time. Additionally, the adverb "accordingly" or "consequently", IMO, should immediately precede the verb "filter" so its clearer that its a cause/effect-- suggest: You can select a portion of it to drill down into the selected time range and accordingly/consequently filter the following flows information.

"Below", according to the IMB Style Guide, isn't ideal. Here's what it says about "below": 
Do not use to indicate a relative location in a document, as in "the information below ". Use "the information later in this section " . Alternatively , you can use "following " to refer to a table , to art, or to a list.

Reply:
Thanks for these feedbacks !